de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/service/impl/LookupDAO.java

```diff
@@ -808,6 +810,17 @@ protected NamePair fetch(final ResultSet rs) throws SQLException
 			final String name = getDisplayName(rs, isActive);
 			final String description = rs.getString(MLookupFactory.COLUMNINDEX_Description);
 
+			String validationMessage;
+			try
+			{
+				validationMessage = rs.getString(MLookupFactory.COLUMNINDEX_ValidationMsg);
+			}
+			catch (Exception e)
```

Comment:
i looked only superficially, but I can't see why we need to take precautions here. 
* why not make the SQL such that the `rs` always contains the validation message?
* if that's not practical, then add a flag or something to tell the iteration if it shall try to get the validationMessage

Reply:
agree